notifications.bs

```diff
@@ -451,7 +451,7 @@ interpreted as a language tag. Validity or well-formedness are not enforced. [[!
 
 <ol>
  <li><p>Wait for any <a for=/ lt=fetch>fetches</a> to complete and <var>notification</var>'s
```

Comment:
Out of scope of this PR: This effectively is await-ing the [fetch steps](https://notifications.spec.whatwg.org/#fetch-steps) in bad way; the fetch steps say the resources are set only when supported but here we are waiting for those resources to be set.

Is there a better wait to wait for a parallel task? 🤔

Reply:
I think we should probably set "image resource" either way (to a decoded resource or to failure or some such). And then we wait for it to not be in its initial state (null?).